jobs/build/aws-ami/Jenkinsfile

```diff
@@ -42,34 +42,38 @@ if ( MOCK.toBoolean() ) {
 }
 
 node(TARGET_NODE) {
-
-    set_workspace()
-
-    stage('build') {
-        try {
-
-            sh(script: '''if [ -d "${WORKSPACE}/openshift-ansible" ]; then 
-                cd ${WORKSPACE}/openshift-ansible ; 
-                git pull; 
-            else
-                git clone https://github.com/openshift/openshift-ansible
-            fi
-
-            cd ${WORKSPACE}
-
-            ''')
-
+    try {
+        set_workspace()
+        def buildlib = null
+        stage('clone') {
+            checkout scm
+            buildlib = load('pipeline-scripts/buildlib.groovy')
+            dir('openshift-ansible') {
+                git 'https://github.com/openshift/openshift-ansible.git'
```

Comment:
What does this do?  I'm assuming it performs a `git clone`.  

The original intention was to check it out if it wasn't there or git pull if it is.

Reply:
Thanks for clarifying.  I imagined that was the original purpose so I changed
it to use the more concise `git` command, which does exactly that.